tests/unit/10_build-env.bats

```diff
@@ -44,6 +44,20 @@ teardown() {
   assert_success
 }
 
+@test "(build-env) buildpack if Dockerfile present" {
```

Comment:
The default app that `deploy_app` attempts to deploy does not contain a `Dockerfile` 
My understanding of this PR is to allow an app that contains a `Dockerfile` be deployed via buildpack if `BUILDPACK_URL` is set. In order to test that it seems like we should instead set `BUILDPACK_URL` to the nodejs buildpack, then call `deploy_app dockerfile` and test for success. Example:

```
@test "(build-env) buildpack deploy with Dockerfile" {
  run dokku config:set --no-restart $TEST_APP BUILDPACK_URL='https://github.com/heroku/heroku-buildpack-nodejs'
  echo "output: "$output
  echo "status: "$status
  assert_success

  run deploy_app dockerfile
  echo "output: "$output
  echo "status: "$status
  assert_success
}
```

Side note: Unless I'm missing something (which happens often depending on the day) this new test, does the same thing as the test below. Definitely correct me if I'm wrong there please. :smile: 

Also, apologies but you'll need to rebase this. We just did a big restructure for commands. I'm happy to take care of it if you'd prefer not to. Just let me know.